README.md

```diff
@@ -1 +1,162 @@
 # ApiTestCase
+[![Build Status](https://travis-ci.org/Lakion/ApiTestCase.svg?branch=master)](https://travis-ci.org/Lakion/ApiTestCase)
+
+**ApiTestCase** is an one-file library that will make your life as a Symfony2 API developer much easier. It extends basic [Symfony2](https://symfony.com/) WebTestCase with some cool features. Thanks to [*PHP-Matcher*](https://github.com/coduo/php-matcher) you can write expected json responses like a gangster(According to php-matcher README.md). [*SymfonyMockerContainer*](https://github.com/PolishSymfonyCommunity/SymfonyMockerContainer) makes it super easy to mock services. This library merges these two great concepts and extends them with a few more lines, to achieve pure awesomeness.
+
+Installation
+------------
+
+Assuming you already have Composer installed globally:
+
+```bash
+$ composer require lakion/api-test-case
+```
+
+Then you have to slightly change your Kernel logic to support SymfonyMockerContainer:
+
+```php
+// app/AppKernel.php
+
+protected function getContainerBaseClass()
+{
+    if ('test' === $this->environment) {
+        return '\PSS\SymfonyMockerContainer\DependencyInjection\MockerContainer';
+    }
+
+    return parent::getContainerBaseClass();
+}
+```
+And its done! ApiTestCase is working with the default configuration.
+
+Configuration reference
+-----------------------
+
+To customize your test suite configuration you can add a few more potions to phpunit.xml:
+```xml
+<php>
+    <server name="KERNEL_DIR" value="/path/to/dir/with/kernel" />
+    <server name="EXPECTED_RESPONSE_DIR" value="/path/to/expected/responses/" />
+    <server name="MOCKED_RESPONSE_DIR" value="/path/to/mocked/responses/" />
+    <server name="OPEN_ERROR_IN_BROWSER" value="true/false" />
+</php>
+```
+ * `KERNEL_DIR` variable contains a path to kernel of your project. If not set, WebTestCase will look for AppKernel in the folder where you have your phpunit.xml file.
+ * `EXPECTED_RESPONSE_DIR` and `MOCKED_RESPONSE_DIR` variables contain paths to folders with expected and mocked responses. `EXPECTED_RESPONSE_DIR` is used when API result is compared with existing json file. `MOCKED_RESPONSE_DIR` should contains files with mocked responses from outside API's. Both variable can have same value but we recommend to keep it separated. If these values aren't set ApiTestCase will try to guess location of responses. It will try to look for the responses in a following folders '../Responses/Expected' and '../Responses/Mocked' relatively located to your controller test class.
+ * `OPEN_ERROR_IN_BROWSER` is a flag which turns on displaying error in a browser window. The default value is false.
+ * `OPEN_BROWSER_COMMAND` is a command which will be used to open browser with an exception.
+ 
+Usage
+-----
+The most basic usage can be achieved by direct comparison of received response and sample file located in  src/YourBundle/Tests/Responses/Expected/hello_world.json. 
```

Comment:
I am not sure you can "achieve basic usage". I imagine the section of usage should show TDD flow.

1. Start from defining your response for a certain request.
2. Put it in a JSON file.
3. Write a test case that sends the request and expects the response from defined in point 1. and contained in file from point 2.
4. Make it red.
5. Make it green.

So generally it is fine, but let's highlight the TDD workflow and maybe show some other than hello world response. Maybe a simple list of products? Only name, sku and price.